config/locales/provider_interface/reasons_for_rejection.yml

```diff
@@ -42,6 +42,9 @@ en:
       vetting_disclosed_information_details: Information revealed by our vetting process makes the candidate unsuitable to work with children - details
       other: Other
       other_details: Safeguarding issues other reason - details
+    cannot_sponsor_visa:
+      title: 'They are not able to sponsor your visa'
```

Comment:
From the design history, this text is "Visa application sponsorship". Did we decide on changing that?

Reply:
Nope, I just put this in early as a placeholder and forgot to change it, good spot!